docs/docs/README.md

```diff
@@ -3,25 +3,127 @@ prev: false
+
+## Use Cases
+
+Pipdream supports use cases from protototype to production and is trusted by 150k+ developers from startups to Fortune 500 companies:
+
+![logos](https://res.cloudinary.com/pipedreamin/image/upload/v1612919944/homepage/logos_kcbviz.png)
+
+The platform processes billions of events and is built and [priced](https://pipedream.com/pricing/) for use at scale. [Our team](https://pipedream.com/about) has built internet scale applications and managed data pipelines in excess of 10 million events per second (EPS) at startups and high-growth environments like BrightRoll, Yahoo!, Affirm and Instacart. 
+
+Our [community](https://pipedream.com/community) uses Pipedream for a wide variety of use cases including:
+
+- Connecting SaaS apps
+- General API orchestration and automation
+- Database automations (learn about [connecting to resources behind a firewall](/workflows/steps/code/nodejs/http-requests/#use-an-http-proxy-to-proxy-requests-through-another-host))
+- Custom notifications and alerting
+- Mobile and JAMstack backends
+- Rate limiting, request smoothing
+- Event queuing and concurrency management
+- Webhook inspection and routing
+- Prototyping and demos
+
+## Open Source
+
+Pipedream maintains an [open source component registry](https://github.com/pipedreamhq/pipedream/) on GitHub. Reuse these components in workflows so you don't need to write boilerplate code for common API integrations, or use existing components as scaffolding to customize and publish your own. You can also [create a PR contribute new or enhanced components](/components/guidelines/#process) via GitHub.
+
+## Contributing
+
+We hope is that by providing a generous free tier, you will not only get value from Pipedream, but you will give back to help us improve the product for the entire community and grow the platform by:
+
+- [Contributing](/components/guidelines/) open source components to our [registry](https://github.com/pipedreamhq/pipedream.) or sharing via your own Github repo
+- Asking and answering questions in our [public community](https://pipedream.com/community/)
+- [Reporting bugs](https://pipedream.com/community/c/bugs/9) and [requesting features](https://github.com/PipedreamHQ/pipedream/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=%5BFEATURE%5D+) that help us build a better product
+- Following us on [Twitter](https://twitter.com/pipedream), staring our [Github repo](https://github.com/PipedreamHQ/pipedream) and subscribing to our [YouTube channel](https://www.youtube.com/c/pipedreamhq)
+- Recommending us to your friends and colleagues
+
+Learn about [all the ways you can contribute](https://pipedream.com/contributing).
+
+## Getting Help
+
+If you have any questions or feedback, please [reach out in our community forum](https://pipedream.com/community).
+
+
+
+<!--
```

Comment:
Do you want to delete this instead of commenting out? We'll have it in previous Git commits if need be.

Reply:
done